.bazelrc

```diff
@@ -1 +1,10 @@
 common --incompatible_disallow_empty_glob
+
+# Use hermetic JDKs for testing and ensure compatibliity with Java 8.
```

Comment:
Are these needed for the rest of this change to work? If not, WDYT about a separate CI job with these flags?

Reply:
Without this, running `bazel test //...` locally would fail if the local JDK is more recent than the JDK in the Java compilation toolchain, which is very likely (I have JDK 23 locally). 

Do you see a downside to setting these flags? Ensuring that CI runs and local runs see the same result seems important for ease of contributing. This wasn't a problem prior to this change as the repo didn't actually contain any Java source files.

Reply:
Yeah, that makes sense, but if it's not too much trouble, doing it in a separate change would be nicer.

Reply:
I split out https://github.com/bazelbuild/rules_java/pull/241